OAuthSDK/YMLoginClient.m

```diff
@@ -225,4 +280,13 @@ - (NSString *)storedAuthToken
     return [[PDKeychainBindings sharedKeychainBindings] objectForKey:YMKeychainAuthTokenKey];
 }
 
+- (NSString *)storedAuthTokenForNetworkPermalink:(NSString *)networkPermalink
```

Comment:
Does this mean that every time when we need a token, we will fetch keychain? It seems not so efficient since accessing keychain might be slow and not stable. Besides, tokens should not be changed during the app lifecycle until user revoke them/logout, could we fetch token only once during the app lifecycle and store it in memory?